packages/core/android/src/test/kotlin/com/datadog/reactnative/DdSdkTest.kt

```diff
@@ -126,13 +135,24 @@ internal class DdSdkTest {
     @Mock
     lateinit var mockChoreographer: Choreographer
```

Comment:
is it still needed if `mockChoreographerCompat` is used now?

Reply:
Yes, because `ChoreographerCompat` is built on top of it:

**ChoreographerCompat.class**
```java
private Choreographer mChoreographer = this.getChoreographer();

private Choreographer getChoreographer() {
    return Choreographer.getInstance();
}

```